dna/tests/src/fixtures.rs

```diff
@@ -0,0 +1,181 @@
+#[cfg(test)]
+pub(crate) mod fixtures {
+    use ::fixt::prelude::*;
+    use hdk::prelude::*;
+    use hdk_crud::{WrappedAgentPubKey, WrappedHeaderHash};
+    use profiles::profile::{Profile, Status as ProfileStatus};
+    use projects::project::goal::crud::{Goal, Hierarchy, Status};
+    use projects::project::{
+        edge::crud::Edge,
+        entry_point::crud::EntryPoint,
+        goal::crud::TimeFrame,
+        goal_comment::crud::GoalComment,
+        goal_member::crud::GoalMember,
+        goal_vote::crud::GoalVote,
+        member::entry::Member,
+        project_meta::crud::{PriorityMode, ProjectMeta},
+    };
+
+    fixturator!(
+      WrappedHeaderHash;
+        constructor fn new(HeaderHash);
+    );
+
+    fixturator!(
+      WrappedAgentPubKey;
+        constructor fn new(AgentPubKey);
+    );
```

Comment:
this comment is not for now, but for later... 

these two themselves should go over into hdk_crud, since that is where these types are defined. 

Also, tell me what you think of this... renaming WrappedHeaderHash to WrappedHeaderAddress and start making consistent at least for our stuff that terminology

Reply:
hmm, the hdk uses HeaderHash and that is the type that is being wrapped changing it to address could be confusing for some? Plus `Hash` uses a few less characters. Although I may be missing some important context as to why Address makes more sense

Reply:
Ah so you feel Hash could be the term to focus and land on. Fair. We will never be able to escape places in function names in the HDK where it says hash but alas thats ok